specs/AppLifecycle/Activation/AppLifecycle Activation.md

```diff
@@ -161,18 +161,25 @@ void RegisterForActivation()
 {
     // Register one or more supported image filetypes, an icon,
     // and zero or more verbs for the File Explorer context menu.
+    string imageFileTypes[3] = { ".jpg", ".png", ".bmp" };
+    string verbs[2] = { "view", "edit" };
     ActivationRegistrationManager::RegisterForFileTypeActivation(
-        new string[]{ ".jpg", ".png", ".bmp" },
-        "MyResources.dll",
-        12,
-        new string[]{ "view", "edit" });
+        imageFileTypes,
+        "MyResources.dll", 0,
+        verbs);
 
-    // Register another set of filetypes, with a different
-    // icon, and no specified verbs.
+    // Register another set of filetypes, falling back to the
+    // default icon, and no specified verbs.
+    string videoFileTypes[3] = { ".mov", ".wmv", ".mp3" };
     ActivationRegistrationManager::RegisterForFileTypeActivation(
-        new string[]{ ".mov", ".wmv", ".mp3" },
-        "MyResources.dll",
-        18);
+        videoFileTypes);
+
+    // Register another set, and specifying the EXE path.
+    string beerFileTypes[1] = { ".ipa" };
+    ActivationRegistrationManager::RegisterForFileTypeActivation(
+        beerFileTypes,
+        "MyResources.dll", 34,
+        "C:\\Program Files\\Contoso\\MyApp.exe");
 
     // Register some URI schemes for protocol activation.
     ActivationRegistrationManager::RegisterForProtocolActivation("foo");
```

Comment:
Does protocol not have a version that takes an EXE?

Reply:
Yes, defined below.